src/app/components/search/DateRangeFilter.js

```diff
@@ -324,9 +326,10 @@ const DateRangeFilter = ({
         getStartDateStringOrNull(),
         getEndDateStringOrNull(),
       ));
-    } else if (valueRangeType === rangeTypes.relative) {
+    } else if (['less_than', 'more_than'].indexOf(valueRangeType) !== -1) {
```

Comment:
I suggest changing this comparison to `['less_than', 'more_than'].includes(valueRangeType)` for better readability

Reply:
Done